app/src/main/res/layout/activity_wallet_details.xml

```diff
@@ -0,0 +1,234 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context="zapsolutions.zap.walletManagement.WalletDetailsActivity">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/type"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="30dp"
+            android:visibility="visible"
+            app:layout_constraintBottom_toTopOf="@id/connectionDataBox"
+            app:layout_constraintTop_toTopOf="parent">
+
+
+            <ImageView
+                android:id="@+id/walletTypeIcon"
+                android:layout_width="0dp"
+                android:layout_height="50dp"
+                android:src="@drawable/ic_remote_black_24dp"
+                android:tint="@android:color/secondary_text_dark"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+
+            <TextView
```

Comment:
Would be nice if we also put some margins at the start and end of this one and then ellipsize it.

![2020-03-29 13_48_20](https://user-images.githubusercontent.com/11649986/77848286-fb7c2500-71c3-11ea-8cf1-1204cdbc0b71.png)

Reply:
Thanks for the feedback, I have just adjusted it.